scripts/dappnode_install.sh

```diff
@@ -289,6 +289,21 @@ installExtraDpkg() {
     fi
 }
 
+add_log_rotate_conf() {
```

Comment:
the log files `.log` are okey to be compressed. But what about the `userActionLog.json` file, this file is consumed by the dappmanager. Can it be consumed even compressed?

Reply:
I would guess no. But idea behind this is that it compresses old files and lets dappmanager consume only latest (in this case less that week old) logs. But we should test thoroughly this to see if it really works.

Reply:
Lets try to have it ready by next devs meeting

Reply:
status?

Reply:
On my machine it behaved fine, @dsimog01 was testing it as well. If it worked for him, we should merge this.